xml/System.IO.Compression/BrotliEncoder.xml

```diff
@@ -201,11 +199,11 @@
         <returns>A number representing the maximum compressed length for the provided input size.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[
-
 ## Remarks
-
-Returns 1 if `inputSize` is 0.
-
+How would it look in the MS Docs Preview.
```

Comment:
Reconfirmed. Empty lines need to be preserved.
https://review.docs.microsoft.com/en-us/dotnet/api/system.io.compression.brotliencoder.getmaxcompressedlength?view=net-5.0&branch=pr-en-us-5213
![image](https://user-images.githubusercontent.com/1175054/104226896-60280980-53fd-11eb-80b2-23c9030e4d7c.png)

Reply:
@safern @gewarren when the time comes where we have DLLs generated by the runtime build alongside their fully documented intellisense XMLs, we should test feeding them to mdoc and see if it is smart enough to determine that _one_ single endline should be rendered as a full `<br />` in MS Docs.